localstack/services/lambda_/lambda_api.py

```diff
@@ -2338,7 +2338,7 @@ def create_code_signing_config():
     signing_profile_version_arns = data.get("AllowedPublishers").get("SigningProfileVersionArns")
 
     code_signing_id = "csc-%s" % long_uid().replace("-", "")[0:17]
-    arn = arns.code_signing_arn(code_signing_id)
+    arn = arns.code_signing_arn(code_signing_id, DEFAULT_AWS_ACCOUNT_ID, AWS_REGION_US_EAST_1)
```

Comment:
For the legacy Lambda provider, again the effort involved to ensure ARNs are correct is substantial hence we resort to default values.